segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java

```diff
@@ -1038,6 +1040,26 @@ public void readTableEntriesDelta(WireCommands.ReadTableEntriesDelta readTableEn
 
     }
 
+    private void truncateIndexSegment(String segment, long offset) {
+        String indexSegment = getIndexSegmentName(segment);
+        try {
+            long indexSegmentOffset = IndexRequestProcessor.locateOffsetForIndexSegment(segmentStore, segment, offset, false);
+            if (indexSegmentOffset == 0) {
+                return;
+            }
+            segmentStore.truncateStreamSegment(indexSegment, indexSegmentOffset, TIMEOUT)
+                    .whenComplete((v, exp) -> {
+                        if (exp != null) {
+                            log.warn("Failed to truncate index segment {} at offset {} due to", indexSegment, indexSegmentOffset, exp);
+                        } else {
+                            log.info("Successfully truncated index segment {} at offset {}", indexSegment, indexSegmentOffset);
+                        }
+                    });
+        } catch (Exception e) {
+            log.warn("Unable to locate offset for index segment {}  for offset {} due to ", indexSegment, offset, e);
```

Comment:
can be changed to : "Unable to locate offset : {} for index segment : {}  due to ", offset, indexSegment, e

Reply:
communicated.